src/data-generator/issue.ts

```diff
@@ -0,0 +1,283 @@
+import fetch from 'node-fetch'
+import { User } from './users'
+
+import { idsToFHIRIds, log, removeEmptyFields } from './util'
+import {
+  AttachmentInput,
+  BirthRegistrationInput,
+  DeathRegistrationInput,
+  LocationType,
+  MarkBirthAsIssuedMutation,
+  MarkDeathAsIssuedMutation,
+  PaymentOutcomeType,
+  PaymentType
+} from './gateway'
+import { omit } from 'lodash'
+import { GATEWAY_GQL_HOST } from './constants'
+import { MARK_BIRTH_AS_ISSUED, MARK_DEATH_AS_ISSUED } from './queries'
+import { differenceInDays } from 'date-fns'
+import { ConfigResponse } from './config'
+import { fetchDeathRegistration, fetchRegistration } from './declare'
+
+export function createBirthIssuingDetails(
+  createdAt: Date,
+  declaration: Awaited<ReturnType<typeof fetchRegistration>>,
+  config: ConfigResponse
+) {
+  const withIdsRemoved = idsToFHIRIds(
+    omit(declaration, ['__typename', 'id', 'registration.type']),
+    [
+      'id',
+      'eventLocation.id',
+      'mother.id',
+      'father.id',
+      'child.id',
+      'registration.id',
+      'informant.individual.id',
+      'informant.id'
+    ]
+  )
+  delete withIdsRemoved.history
+
+  const completionDays = differenceInDays(
+    createdAt,
+    new Date(declaration.child?.birthDate!)
+  )
+
+  const paymentAmount =
+    completionDays < config.config.BIRTH.REGISTRATION_TARGET
+      ? config.config.BIRTH.FEE.ON_TIME
+      : completionDays < config.config.BIRTH.LATE_REGISTRATION_TARGET
+      ? config.config.BIRTH.FEE.LATE
+      : config.config.BIRTH.FEE.DELAYED
+  log(
+    'Collecting certification payment of',
+    paymentAmount,
+    'for completion days',
+    completionDays
+  )
```

Comment:
From now on we should only need this for issuing. So could you please remove it from the certify flow?